tally_ho/apps/tally/views/data/center_list_view.py

```diff
@@ -60,7 +75,24 @@ class CenterListView(LoginRequiredMixin,
 
     def get(self, *args, **kwargs):
         tally_id = kwargs.get('tally_id')
+        region_id = kwargs.get('region_id', None)
```

Comment:
the default is already `None`, I think it's fine to not pass it here and the below 2

Reply:
Makes sense

Reply:
Done